docs/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Minor change to enable perf tests to run in CI (OSIDB-2447)
 - Allow editing flaws without affects in NEW state (OSIDB-2452)
 - Fixed read replica to perform HTTP requests as atomic transactions (OSIDB-2585)
+- Removed db routing 
```

Comment:
I am checking the changelog. The DB routing is on master said to be added in 3.7.0 but when I check out to 3.7.0 tag the item is not present in the CHANGELOG. So it is incorrectly added here under 3.7.0 and should under Unreleased instead. Then we would announce adding and removing it in a single release which does not make much sense. I suggest removing it from here and from 3.7.0 below together with the replica fix (hope this is all related) as part of this change basically reverting the CHANGELOG so the users get clearer information.

Reply:
I think that Jim's change is correct. The release was made from the `merge-3.7.0-to-master` branch, see CHANGELOG in #499.

Reply:
@JimFuller-RedHat @osoukup Sorry for the merge interruption, but could you please check the above?

Reply:
as discussed with @osoukup - we are not adding and removing db routing in a single release - and we never released db routing in last release (was a mistake).

Reply:
Jitka is right. Yes, the line `Add db routing taking advantage of write and read pg replicas` was included in 3.7.0 already. And `git blame` and searching the commit f529baef in `git log -v --source  --log-size  --notes --graph --name-status --show-signature` shows that it's been added before 3.7.0.

Reply:
Ondrej is right in the end :). That change was not released, CHANGELOG in #499 is incorrect and Ondrej already fixed the release notes.